cmake/CCCLConfigureTarget.cmake

```diff
@@ -0,0 +1,57 @@
+set(CCCL_EXECUTABLE_OUTPUT_DIR "${CCCL_BINARY_DIR}/bin")
+set(CCCL_LIBRARY_OUTPUT_DIR "${CCCL_BINARY_DIR}/lib")
+
+# Setup common properties for all test/example/etc targets.
+function(cccl_configure_target target_name)
+  set(options)
```

Comment:
Can we drop those? AFAIC they are unused

Reply:
I like to leave those as stubs, as they document the parameters for `cmake_parse_arguments` when/if I need to add more. This is a common practice when using `cmake_parse_arguments`.